src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -0,0 +1,94 @@
+{#**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<tr id="orderProduct_{{ product.orderDetailId }}">
+  <td class="cellProductImg">
+    {% if product.imagePath %}
+      <img src="{{ product.imagePath }}" alt="{{ product.name }}">
+    {% endif %}
+  </td>
+  <td class="cellProductName">
+    <a href="{{ path('admin_product_form', {'id': product.id}) }}">
+      <p class="mb-0 productName">{{ product.name }}</p>
+      {% if product.reference %}
+        <p class="mb-0 productReference">
+          {{ 'Reference number:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+          {{ product.reference }}
+        </p>
+      {% endif %}
+      {% if product.supplierReference is not empty %}
+        <p class="mb-0 productSupplierReference">
+          {{ 'Supplier reference:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+          {{ product.supplierReference }}
+        </p>
+      {% endif %}
+    </a>
+  </td>
+  <td class="cellProductUnitPrice">{{ product.unitPrice }}</td>
+  <td class="cellProductQuantity">
+    {% if product.quantity > 1 %}
+      <span class="badge badge-secondary rounded-circle">{{ product.quantity }}</span>
+    {% else %}
+      {{ product.quantity }}
+    {% endif %}
+
+    <div class="d-none js-product-quantity">
+      <input type="text" value="{{ product.quantity }}" class="form-control">
+    </div>
+  </td>
+  <td class="cellProductLocation">{{ product.location }}</td>
+  <td class="cellProductAvailableQuantity">{{ product.availableQuantity }}</td>
+  <td class="cellProductTotalPrice">{{ product.totalPrice }}</td>
+  {% if not orderForViewing.delivered %}
+    <td class="text-right cellProductActions">
+      <a href="javascript:void(0);"
```

Comment:
This is overly complicated for no action :P Can't you just remove the href everywhere? Besides as I suggested in another comment a button would be much more adapted here and it should display the same way as an `a` tag -> `<button type="button" class="btn">`

Oh unless you need the `hover` effect related to `tooltip-link` (color change) I noticed this effect was not present with the `.btn .material-icons` as well in my Currency PR, but I think it would be a great addition (to confirm with Scott or Tristan)

But anyway this `javascript:void(0);` is really ugly and should be removed everywhere